libcxx/include/__memory/allocator_traits.h

```diff
@@ -300,11 +300,7 @@ struct _LIBCPP_TEMPLATE_VIS allocator_traits
         __enable_if_t<!__has_construct<allocator_type, _Tp*, _Args...>::value> >
     _LIBCPP_INLINE_VISIBILITY _LIBCPP_CONSTEXPR_SINCE_CXX20
     static void construct(allocator_type&, _Tp* __p, _Args&&... __args) {
-#if _LIBCPP_STD_VER >= 20
```

Comment:
Same question as below.

Are we required to use `construct_at` here? If not, it seems to be more expensive, and causes arguably poorer diagnostic messages.

Reply:
This is for constexpr-friendliness IIRC.